### PR TITLE
Digitise chapters 1, 3-12

### PR DESCRIPTION
**Pull request by KlaraSkinner**

### CODE REVIEW

File: palliative-care-for-children/text/09.md [hunk 1]
```diff
@@ -1,531 +1,424 @@
...
-**\[Box\] Shortness of breath is a common symptom in children who need palliative care.**
+> Shortness of breath is a common symptom in children who need palliative care.
 
-## What are the common causes of shortness of breath?
+### 9-9 What are the common causes of shortness of breath?
 
 There are many underlying conditions or causes for shortness of breath which will require specific management or treatment. It is therefore important to assess the child to determine the possible cause. Common causes in children’s palliative care are related to advanced cancer, AIDS, anaemia, chest infections, heart conditions, neuromuscular diseases, stress, anxiety and treatments such as post radiation fibrosis.
 
-**\[Box\]There may be many underlying causes of shortness of breath so it is important to assess the child to determine the correct treatment.**
+>There may be many underlying causes of shortness of breath so it is important to assess the child to determine the correct treatment.
```
Comment (LaurenEllwood) [inline palliative-care-for-children/text/09.md]: Insert space before 'There'

File: palliative-care-for-children/text/10.md [hunk 2]
```diff
@@ -1,875 +1,666 @@
...
--   Give the child choices as to what and when they eat and drink
-
--   Maintain good oral hygiene and mouth care.
-
-*Note: Increasing calorie intake will not increase weight gain and improve the quality of life in the advanced stages of disease. Often this extra intake of foods and fluids puts an added burden on, without benefit to, the body if it is shutting down.*
-
-**\[Box\] It is vital to explain to parents that they are not starving or causing harm if their child does not want to eat.**
-
-## Are there any medications that can help increase appetite?
+### 10-25 Are there any medications that can help increase appetite?
```
Comment (LaurenEllwood) [inline palliative-care-for-children/text/10.md]: Section numbering is incorrect. This should be 10-24. The rest of the chapter will also need to be renumbered.

File: palliative-care-for-children/text/10.md [hunk 3]
```diff
@@ -1,875 +1,666 @@
...
+2. **Friction:** This is when the child’s skin is pulled or repeatedly rubbed against a surface such as a mattress, bedding or other equipment. This occurs when a child slides down in the bed or chair or by incorrect moving and handling.
+3. **Moisture:** If areas of the skin remain moist for extended periods of time they become more prone to breakdown and the likelihood of a pressure sore developing. Children who are incontinent or infants who lie in wet nappies are at risk.
+4. **Previous skin damage:** If the child has had a pressure ulcer in the past, the skin will be prone to further damage.
 
-2.  **Friction:** This is when the child’s skin is pulled or repeatedly rubbed against a surface such as a mattress, bedding or other equipment. This occurs when a child slides down in the bed or chair or by incorrect moving and handling.
+> > Immobility, local pressure, friction and moisture are the main causes of pressure sores.
```
Comment (LaurenEllwood) [inline palliative-care-for-children/text/10.md]: Remove second indentation.

File: palliative-care-for-children/text/12.md [hunk 4]
```diff
@@ -1,308 +1,272 @@
...
+3. **To adjust to the world without the deceased:** There are three adjustments to be made, namely external adjustments, internal adjustments and spiritual adjustments. In other words, how the death affects your everyday function, how it affects your sense of self and how it affects your beliefs, values and assumptions about the world.
+4. **To find an enduring connection with the deceased while starting a new life:** The bereaved person creates a balance between remembering the person who died and finding a way to continue to live a meaningful life without them.
 
-4.  **To find an enduring connection with the deceased while starting a new life**: The bereaved person creates a balance between remembering the person who died and finding a way to continue to live a meaningful life without them.
+> Warden’s four tasks of mourning are accepting the reality of the loss, processing the emotional pain of grief, adjusting to life without the loved one, and finding a way to remember them.
```
Comment (LaurenEllwood) [inline palliative-care-for-children/text/12.md]: Please change 'Warden's' to 'Worden's'.

File: palliative-care-for-children/text/12.md [hunk 5]
```diff
@@ -312,550 +276,409 @@ Parents must be encouraged to speak in an open, honest and age appropriate way w
...
-
-Leroy and Shirley’s daughter, Bianca, died of leukaemia two months ago at the age of 6. Shirley is still very upset, crying most of the day and does not sleep well at night. Hannah, who is 13-months-old, has become increasingly irritable and clingy, and John, who is-4 years-old, keeps asking if Bianca will be back to play with him. During the grief counselling session, Shirley says that she still cannot believe that this has happened, and that she just wants to wake up from this nightmare. She admits that she thinks Hannah is being naughty and she does not know how to handle her clinginess. She also says she doesn’t know how to answer John’s question so keeps telling him that Bianca has gone away and won’t be coming back.
-
-1.  **What types of losses have Leroy and Shirley experienced?**
+### 1. What types of losses have Leroy and Shirley experienced?
 
 They have experienced the primary loss (death) of their daughter and with this loss they will have experienced many secondary losses. For example, the loss of a future with her, the loss of the dreams and hopes they may have had for her,
 
 the loss of seeing her go to school and the loss of her being the eldest child.
```
Comment (LaurenEllwood) [inline palliative-care-for-children/text/12.md]: Remove line break in question 1.

File: palliative-care-for-children/text/12.md [hunk 6]
```diff
@@ -312,550 +276,409 @@ Parents must be encouraged to speak in an open, honest and age appropriate way w
...
+### 2. How could you support the family at the time of George’s death?
 
-1.  **What other rituals can the siblings participate in if they do not attend the funeral? **
+Remembering that the family will always remember what happened at the time of their child’s death, the family should be treated with great compassion and given as much support as possible.
+- Acknowledge their loss with sincerity and reassure them that you are there to support them and to answer any questions they have. Prepare them for what might happen over the next period of time.
+- Provide a private space for the family and allow them to have as much time as they need to spend with their baby.
+- Encourage them to create memories. This could include taking a lock of hair from the baby, making hand or footprints and taking photographs with the baby.
+- At the right time, offer practical support, such as asking who you could contact on their behalf, calling the funeral home and ensuring that they are able to get home safely from the hospital.
+- Refer them to someone who will be able to give them more medical information about Trisomy 18 and the risk of any other children being affected by the same condition
```
Comment (LaurenEllwood) [inline palliative-care-for-children/text/12.md]: Add fullstop to the last bullet point.

File: palliative-care-for-children/text/01.md [hunk 7]
```diff
@@ -1,493 +1,448 @@
...
 In practise, however, it often depends on the individual child’s age, maturity, stage of development and illness as well as the wishes of the parents and the guidelines of the local health facility.
 
-1.  **What is children’s palliative care?**
+### 1-5 What is children’s palliative care?
 
-**The World Health Organization defines palliative care for children as follows:**
+The World Health Organization defines palliative care for children as follows:
 
-**Palliative care for children represents a special, closely related field to adult palliative care. Palliative care for children is the active total care of the child's body, mind and spirit, and also involves giving support to the family.**
+Palliative care for children represents a special, closely related field to adult palliative care. Palliative care for children is the active total care of the child's body, mind and spirit, and also involves giving support to the family.
```
[the remark this answers is not in the document]
Comment (KlaraSkinner) [reply palliative-care-for-children/text/01.md]: Adding a new class for quotes in a separate PR.

**Pull request merged**